bioframe/core/arrops.py

```diff
@@ -296,8 +312,11 @@ def overlap_intervals(starts1, ends1, starts2, ends2, closed=False, sort=False):
 
     starts1 = np.asarray(starts1)
     ends1 = np.asarray(ends1)
+    starts1, ends1 = get_pseudo_segment(starts1, ends1)
```

Comment:
Similarly, it'd be nice to add a comment here that would explain the problem with the original solution and the trick that you came up with to solve it.

Reply:
Noted, will do that.